tests/test_odgovlt.py

```diff
@@ -176,12 +176,13 @@ def test_OdgovltHarvester(app, db, mocker):
     result = harvester.fetch_stage(obj2)
     assert obj2.errors == []
     assert result
-    create_or_update = harvester.import_stage(obj1)
+    create_or_update, package_dict = harvester.import_stage(obj1)
     assert create_or_update
     create_or_update = harvester.import_stage(obj2)
     assert create_or_update
     assert obj1.package_id
     assert obj2.package_id
+    assert 'owner_org' in package_dict
```

Comment:
That is not exactly the test we need. Previously I got error about missing `owner_org` while running `harvester run_test` command. This command does a lot more checks comparet to what you have here. So to make sure, that no more errors comes for the higher level that we currently don't test, we neet to update tests, to also cover higher level parts.

So basically you need to run `harvester run_test` command logic for the tests. In order to do that, you need to make yourself familiar with `harvester run_test` command code, and decide, how this command could be executed from tests. You can find coomand code here:

https://github.com/ckan/ckanext-harvest/blob/master/ckanext/harvest/commands/harvester.py#L417